console-subscriber/src/lib.rs

```diff
@@ -16,13 +16,69 @@ use tracing_core::{
     subscriber::{self, Subscriber},
     Metadata,
 };
-use tracing_subscriber::{layer::Context, registry::LookupSpan, Layer};
+use tracing_subscriber::{fmt, layer::Context, registry::LookupSpan, EnvFilter, Layer};
 
 mod aggregator;
 use aggregator::Aggregator;
 mod builder;
 pub use builder::Builder;
 
+/**
+Starts the console subscriber server on its own thread
+
+This function represents the easiest way to get started using
+tokio-console.
+
```

Comment:
these docs should probably state explicitly that this function also sets the default `tracing` subscriber (and maybe include a link to the `tracing` docs?). otherwise, i can see people being surprised if they try to set their own subscriber and it fails because the default subscriber is already set.

Reply:
addressed in  f2b6de5